templates/js/api-plugin-from-scratch-bearer/README.md

```diff
@@ -0,0 +1,85 @@
+# Overview of the Copilot Plugin template
+
+## Build a Copilot Plugin from a new API with Azure Functions
+
```

Comment:
I suggest updating the title to include authentication mechanism here: `Build a Copilot Plugin from a new API authenticated by API Key with Azure Function`